requirements.txt

```diff
@@ -7,4 +7,4 @@ mapbox-vector-tile
 pyclipper
 # Use a patch fork, for unmaintened rauth, https://github.com/litl/rauth/issues/185
 git+git://github.com/hendi/rauth.git@37615b738a944684bb02ea1c9ea5f3bbde84f0e7
-bottle == 0.12.13
+bottle == 0.12.18
```

Comment:
Does it belong in this commit?
There might be more trials to be done if we update bottle version.

Reply:
> Does it belong in this commit?
> There might be more trials to be done if we update bottle version.

It's intended, due to:

https://github.com/osm-fr/osmose-frontend/issues/248

This is not exactly TESTED, I'm trying to make my docker setup working in the meanwhile...

Reply:
Ok for getargspec to getfullargspec. Please just remove bottle == 0.12.18 from this PR.